for_streamlit_only/data_customisation.py

```diff
@@ -41,7 +42,9 @@ def get_deg_win(chart_type, spectra_conversion_type, cols_right, df_columns):
                 adjust_plots_globally = st.radio(
                     "Adjust all spectra or each spectrum?",
                     ('all', 'each'), index=0)
-                
+    
+                df = for_streamlit_only.utils.trim_spectra(df)
```

Comment:
Nie, to nie w tym miejscu 😠 
get_deg_win ma się zajmować deg'iem i win'em

To zasługuje na osobne wywołanie bezpośrednio w vis.py.  Tylko daj tam komentarz, że trim_spectra rysuje własny suwak, żeby było widać na pierwszy rzut oka ;)

Reply:
no, ale jak tak zrobię, to wtedy będzie to globalne, a nie dla danego widma. 
No i wtedy nie będę chyba wstanie tego wrzucić do tej prawej kolumny?